code/go/0chain.net/smartcontract/storagesc/allocation_eventdb.go

```diff
@@ -242,7 +240,7 @@ func (sa *StorageAllocation) buildDbUpdates() *dbs.DbUpdates {
 			"read_price_max":            sa.ReadPriceRange.Max,
 			"write_price_min":           sa.WritePriceRange.Min,
 			"write_price_max":           sa.WritePriceRange.Max,
-			"challenge_completion_time": int64(sa.ChallengeCompletionTime),
+			"challenge_completion_time": int64(getMaxChallengeCompletionTime()),
```

Comment:
There is now no  `challenge_completion_time` in the db, it was removed in [dbs/event/allocation.go above](https://github.com/0chain/0chain/pull/1706/files#diff-7fcb1d502e80f35327fc6dea8dfa929899e201d7eaba78bc90a71fe4eefd8dc5L33). So can remove this line.